web/src/helpers/dom.js

```diff
@@ -0,0 +1,7 @@
+export function modifyDOM(ref, selector, callback) {
```

Comment:
Maybe this function should only be in FAQ if it's only used there and it's supposed to be removed in the future?

Reply:
Yeah good point, I used it temporarily in tooltips but now it's just used in FAQ again so I reverted it in https://github.com/tmrowco/electricitymap-contrib/pull/2276/commits/bb4e3f80792ec73aef1f56017089a72b49314547 :)